final_project_schema.sql

```diff
@@ -0,0 +1,190 @@
+                        ('6bc88650-535b-4c44-9a7a-11ebe27bab83', 'Honda', 'Civic', '2018'),
+                        ('cebef27b-8de6-42c5-bbc9-a2e8158ab3ae', 'Ford', 'F-150', '2016'),
+                        ('da4ad3d4-2d51-47bd-927f-ba9a2812ae4f', 'Chevrolet', 'Silverado', '2023'),
+                        ('338763f1-cdd4-43e7-9c69-8f248df1abbf', 'Nissan', 'Leaf', '2024'),
+                        ('4c964948-caad-4fcd-b917-52af70367ca5', 'Volkswagen', 'Beetle', '2020');
+
+INSERT INTO drive VALUES
+                      ('8574e1d5-7369-4f23-8c5f-0f53d8977303', '6a6a459f-4986-4f23-b9f5-a8ec1923ef6d', 20.4, 40, 'a great drive!', 'https://img.freepik.com/free-photo/luxurious-car-parked-highway-with-illuminated-headlight-sunset_181624-60607.jpg', NOW(), 'asico'),
+                      ('3f16c4e4-e8d0-4175-84f3-53ee91923ec1', '6a6a459f-4986-4f23-b9f5-a8ec1923ef6d', 19.7, 39, 'a great drive!', 'https://img.freepik.com/free-photo/luxurious-car-parked-highway-with-illuminated-headlight-sunset_181624-60607.jpg', NOW(), 'asico'),
+                      ('5c971fbf-7210-4f9e-8358-4e6d54defc88', 'ffa2b0b9-efd5-4dc9-b947-d9a25af99692', 10.4, 23, 'a great drive!', 'https://img.freepik.com/free-photo/luxurious-car-parked-highway-with-illuminated-headlight-sunset_181624-60607.jpg', NOW(), 'stanker'),
+                      ('457fdf98-de29-421b-8ab6-fbea9d780cc2', 'cebef27b-8de6-42c5-bbc9-a2e8158ab3ae', 22.2, 58, 'a great drive!', 'https://img.freepik.com/free-photo/luxurious-car-parked-highway-with-illuminated-headlight-sunset_181624-60607.jpg', NOW(), 'svasire4'),
+                      ('5282a1ac-b818-4cac-b3f3-5fca6732cd15', 'cebef27b-8de6-42c5-bbc9-a2e8158ab3ae', 12.7, 36, 'a great drive!', 'https://img.freepik.com/free-photo/luxurious-car-parked-highway-with-illuminated-headlight-sunset_181624-60607.jpg', NOW(), 'svasire4'),
+                      ('1dad8038-ac60-42b3-80f1-ff10d0762778', '6bc88650-535b-4c44-9a7a-11ebe27bab83', 37.5, 58, 'a great drive!', 'https://img.freepik.com/free-photo/luxurious-car-parked-highway-with-illuminated-headlight-sunset_181624-60607.jpg', NOW(), 'ccuartas'),
+                      ('bb6fcd45-d54d-404f-89a7-f01e60fb0bf3', 'ffa2b0b9-efd5-4dc9-b947-d9a25af99692', 15.6, 26, 'a great drive!', 'https://img.freepik.com/free-photo/luxurious-car-parked-highway-with-illuminated-headlight-sunset_181624-60607.jpg', NOW(), 'ccuartas'),
+                      ('e5aec1d5-4a8a-4478-b2f0-e573244a6b4b', 'da4ad3d4-2d51-47bd-927f-ba9a2812ae4f', 55.2, 115, 'a great drive!', 'https://img.freepik.com/free-photo/luxurious-car-parked-highway-with-illuminated-headlight-sunset_181624-60607.jpg', NOW(),'skiser18'),
+                      ('b6581a62-a5ac-4ec7-a043-006724940583', '4c964948-caad-4fcd-b917-52af70367ca5', 32.1, 42, 'a great drive!', 'https://img.freepik.com/free-photo/luxurious-car-parked-highway-with-illuminated-headlight-sunset_181624-60607.jpg', NOW(),  'mwilki31');
```

Comment:
This is just personal preference but wouldn't it be better to add the photos to our static folder instead of a link. I think it would just be easier to read.

besides that it looks good as is!

Reply:
Professor mentioned that the photos would be stored as a link in the cloud later on.  I think he will cover how to do that soon and that will be a sprint 4 problem to successfully implement.  He said not to store the photos themselves in the database.